inc/authldap.class.php

```diff
@@ -1194,6 +1201,16 @@ function rawSearchOptions() {
          'autocomplete'       => true,
       ];
 
+      $tab[] = [
+         'id'                 => '32',
+         'table'              => $this->getTable(),
+         'field'              => 'timeout',
+         'name'               => __('Timeout'),
+         'massiveaction'      => false,
+         'datatype'           => 'number',
```

Comment:
```suggestion
         'datatype'           => 'number',
         'unit'               => 'second',
```

Reply:
Not sure about that. If we allow `0` value, it should not display "0 second".

Reply:
done